autoInstall.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+import sys
+import os
+
+def install(package):
+    subprocess.check_call([sys.executable, "-m", "pip", "install", package])
+    
+os.chdir("Tuxemon-development/Tuxemon-development")
+
+req = open("requirements.txt", "r")
+reqtemp = req.read()
+reqlist = reqtemp.split("\n")
+print(reqlist)
+
+
+listpos = 0
+while True:
```

Comment:
What's the benefit of this loop Vs the pip command in the readme?

Reply:
from what I can see, nothing really. this is my first github commit ever and just did it without much thought.